site/cypress/e2e/Cypress-Feature/office_hours_queue.spec.js

```diff
@@ -132,7 +132,22 @@ describe('test office hours queue', () => {
         editAnnouncement('');
         cy.get('[data-testid="announcement"]').should('not.exist');
         cy.get('[data-testid="popup-message"]').should('contain', 'Updated announcement');
-        // diable and delete all queue
+        // disable and delete all queue
         disableQueue();
     });
+    it('Enabling push and sound notifications as Instructor', () => {
+        switchUser('instructor');
+        cy.get('[data-testid="push-notification-switch"]').should('exist');
+        cy.get('[data-testid="sound-notification-switch"]').should('exist');
+        cy.window().its('notifications_enabled').should('equal', false);
+        cy.window().its('isAudibleAlertEnabled').should('equal', false);
+        cy.get('[data-testid="push-notification-switch"]').click();
+        cy.get('[data-testid="sound-notification-switch"]').click();
+        cy.window().its('notifications_enabled').should('equal', true);
+        cy.window().its('isAudibleAlertEnabled').should('equal', true);
```

Comment:
How hard would it be to make these consistent, either snake case or camel case? If it's too hard, I don't think it's worth it, but if it would be a simple refactor, it would make the codebase a little more consistent.

Reply:
That's fair. Refactored in [6ea291f](https://github.com/Submitty/Submitty/pull/10564/commits/6ea291fe6d48c8b074a318edaeaa792ec9d25a5b)